### ISSUE TITLE
Enabling with code analysis on a 3.1 project also requires setting AnalysisLevel

### ISSUE THREAD

**Issue by rynowak** [dotnet/docs#21835]
Either there's a product bug in 5.0.100 **or** the docs are missing some detail about pre-.NET-5 projects.

My project is using netcoreapp3.1 and opting in to code analysis via `<EnableNETAnalyzers>true</EnableNETAnalyzers>` has no effect. I had to also set `<AnalysisLevel>default</AnalysisLevel>` to see analysis start working.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f8906b2c-b984-0427-d064-829e7ea2fc55
* Version Independent ID: 128f9903-971d-6a66-a2cb-9b12acbc55e7
* Content: [Code analysis in .NET](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/overview)
* Content Source: [docs/fundamentals/code-analysis/overview.md](https://github.com/dotnet/docs/blob/master/docs/fundamentals/code-analysis/overview.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @gewarren
* Microsoft Alias: **gewarren**

**Comment by gewarren** [comment]
@rynowak I just tried this out and I didn't have to add the `AnalysisMode` property (I assume you meant that instead of `AnalysisLevel`). I created a .NET Core 3.1 console app in Visual Studio and another one using `dotnet new`. In both cases, all I had to do was add `<EnableNETAnalyzers>true</EnableNETAnalyzers>` to the project file.

Note that there are only a handful of rules enabled by default, which is why it might have appeared that code analysis wasn't enabled. I tested it out with this code, which triggers one of the rules that's enabled by default.

```csharp
// CA2013
int int1 = 1, int2 = 1;
Console.WriteLine(Object.ReferenceEquals(int1, int2));
```
